Sources/CassandraClient/Data.swift

```diff
@@ -283,6 +308,16 @@ extension CassandraClient {
             cass_value_is_null(self.rawPointer) == cass_true
         }
     }
+
+    /// A reusable page token that can be used by `Statement` to resume querying
+    /// at a specific position.
+    public struct PagingStateToken {
+        public let token: [UInt8]
```

Comment:
Given the many warnings about these things being unsafe to share with external users or to create from arbitrary data, should we remove the public fields and public init? That would make this type opaque, and so ensure no inappropriate sharing.

Reply:
For my use case I actually do want to use externally. Taking the appropriate safety measures like the Java client takes.

Reply:
Some of the warnings come from the C library which also exposes this API. So I think the question is whether or not this library sits closer to the C implementation or offers more high level features like the Java client.

If the latter, then we may need to accept increasing the breadth of dependencies (such as on Swift Crypto in this case).